### ISSUE TITLE
Why why why why ...? 

### ISSUE THREAD

**Issue by ZatarraTR** [MAFINS/MenyooSP#226]
You persistently avoid updating.
Will these people beg you?
Is it a crime to ask this beautiful app you made to be updated?
If you are not going to update it, close this page ..
We know that we have said goodbye to Menyoo now. 
I am bored and will not use it anymore.

**Comment by ReadyP1ayerTwo** [comment]
> You persistently avoid updating.

What you fail to realize is mafins doesn't update it anymore himself. The community updates it. Infact, it got updated within a few recent months ago. Instead of people such as yourself complaining about bugs, You could make yourself useful and use your time more productively by fixing these bugs you people oh so endlessly complain about.

With that being said; i use menyoo all of the time with little to no issue.

**Comment by ZatarraTR** [comment]
Are you a lawyer?
You use it smoothly, why do those here complain?

**Comment by frafflg** [comment]
> 
> 
> > You persistently avoid updating.
> 
> What you fail to realize is mafins doesn't update it anymore himself. The community updates it. Infact, it got updated within a few recent months ago. Instead of people such as yourself complaining about bugs, You could make yourself useful and use your time more productively by fixing these bugs you people oh so endlessly complain about.
> 
> With that being said; i use menyoo all of the time with little to no issue.

so there is another group or page on internet were the recent updates for this trainer are getting created , can you share the link here so we can access that page ?

**Comment by ReadyP1ayerTwo** [comment]
> > > You persistently avoid updating.
> > 
> > 
> > What you fail to realize is mafins doesn't update it anymore himself. The community updates it. Infact, it got updated within a few recent months ago. Instead of people such as yourself complaining about bugs, You could make yourself useful and use your time more productively by fixing these bugs you people oh so endlessly complain about.
> > With that being said; i use menyoo all of the time with little to no issue.
> 
> so there is another group or page on internet were the recent updates for this trainer are getting created , can you share the link here so we can access that page ?

Let me re-phrase; Recently enough considering the updates have been left to the community. 
https://github.com/MAFINS/MenyooSP/releases

**Comment by ReadyP1ayerTwo** [comment]
> Are you a lawyer?
> You use it smoothly, why do those here complain?

Probably because people like to complain and do nothing to solve the issue for themselves. People expect things to be spoon fed to them.

**Comment by frafflg** [comment]
> 
> 
> > > > You persistently avoid updating.
> > > 
> > > 
> > > What you fail to realize is mafins doesn't update it anymore himself. The community updates it. Infact, it got updated within a few recent months ago. Instead of people such as yourself complaining about bugs, You could make yourself useful and use your time more productively by fixing these bugs you people oh so endlessly complain about.
> > > With that being said; i use menyoo all of the time with little to no issue.
> > 
> > 
> > so there is another group or page on internet were the recent updates for this trainer are getting created , can you share the link here so we can access that page ?
> 
> Let me re-phrase; Recently enough considering the updates have been left to the community.
> https://github.com/MAFINS/MenyooSP/releases

oh ok sorry sorry , i didn't understood that you are a motherfucking retarded shit , a miserable dick head that isn't of any help , this "do it yourself" is just fucking egoism , it is what assholes tell and do to others, if we could do that ourselves we would already did that , but since we aren't all hackers then we rightly ask to the person who created the trainer , you are the wrong one here , not the person who rightly demanded for this trainer to be updated , just shut the fuck up , oh right , i forgot that assholes like you think they are always right , so when someone insult them because they act egoistic and idiotic they get all surprised when they finally find someone who insult them , well here is the thing , your bullshit , your egoism is what  caused you to get my insults , you are wrong and you keep insist despite this , mafins or manyoo created this trainer so he must update and keep developing it , your bullshit about the updates left to the community is just a less rude way to say "i don't give a fuck do it yourself" , is like you tried to make it sound like an encouragement , but you perfectly understand that none of us can update that trainer , so basically this is just not giving a damn about left some people with an unsolved problem , and to make thing even more insulting for us you even have the fucking nerve to complain , just shut the fuck up you useless dick head , fuck you , and keep in your fucked up mind that none is fully independent , not even you , if you need help for something that for some reasons can't achieve all by yourself then you need help , when that happen you will just deserve to get the same treatment that you inflicted on us here , dick , and i know that retarded assholes like you need things to be repeated many times otherwise you don't understand\pretend you didn't understood: I INSULTED YOU BECAUSE YOU INSULTED ME AND THE OTHER PERSON IN THIS DISCUSSION , IT WASN'T SOME RANDOM INSULTS , YOU FUCKING ASKED FOR THEM BECAUSE OF THE OFFENSIVE BULLSHIT YOU SAID , all clear ? it is clear enough for your handicapped little brain ? when you are rude to people , when you are a fucking pain in the ass expect at least this

**Comment by frafflg** [comment]
> 
> 
> Are you a lawyer?
> You use it smoothly, why do those here complain?

don't mind that handicapped shit , he is only a motherfucking shit head with a face made of shit trowed here to bother us , your claims are completely legit , don't worry the trainer will be updated , it is a matter of common sense after all

**Comment by ZatarraTR** [comment]
> > Are you a lawyer?
> > You use it smoothly, why do those here complain?
> 
> don't mind that handicapped shit , he is only a motherfucking shit head with a face made of shit trowed here to bother us , your claims are completely legit , don't worry the trainer will be updated , it is a matter of common sense after all

**Thank you for your support.**

**Comment by CptLeetsy** [comment]
> mafins or manyoo created this trainer so he must update and keep developing it`

No no no and no... Mafins doesnt have to update it, i got so annoyed by your attitude that i made an account just answer you.
Menyoo is (at least for me) still working very fine. As far as i know, Mafins won't update this trainer, and it's just fine, he doesn't have any obligations to do it. 
It's his work, and if he desires to stop working on it, then it's his rights, whether you like it or not.

And inbefore you spit your 2IQ hate on me, i'm not a programmer by any means, i use the last version of Menyoo like you do.
ReadyP1ayerTwo simply explained to you that this mod was only updated by the community. He never said that he was able to update it, or that he updated it for himself.
And let me tell you, if he had updated it just for himself, then it's as well, his own damn rights to keep it for himself, call him sellfish or all sorts of names if you want, he doesn't owe you or the community anything, neither does Mafins.

So if you have issues with Menyoo, please find a new trainer that is up to date, because it's highly probably that Menyoo won't see any updates in a while.

**Comment by ZatarraTR** [comment]
Then the Menyoo Programmer has to announce that it's over.
People here should not be in expectation.
Updates by the community are not enough.
It contains many errors.
Therefore, shouldn't the programmer say that I will no longer support it?

**Comment by CptLeetsy** [comment]
I don't know if Mafins officially said that he was done with Menyoo, i never saw anything regarding that right here. It was just made clear by the fact the trainer stopped getting udpates at some point.
So yes, Mafins could have (in the case he didn't) said that, Menyoo was done. 

**But he isn't obligated to do it**

**Comment by frafflg** [comment]
> 
> 
> > mafins or manyoo created this trainer so he must update and keep developing it`
> 
> No no no and no... Mafins doesnt have to update it, i got so annoyed by your attitude that i made an account just answer you.
> Menyoo is (at least for me) still working very fine. As far as i know, Mafins won't update this trainer, and it's just fine, he doesn't have any obligations to do it.
> It's his work, and if he desires to stop working on it, then it's his rights, whether you like it or not.
> 
> And inbefore you spit your 2IQ hate on me, i'm not a programmer by any means, i use the last version of Menyoo like you do.
> ReadyP1ayerTwo simply explained to you that this mod was only updated by the community. He never said that he was able to update it, or that he updated it for himself.
> And let me tell you, if he had updated it just for himself, then it's as well, his own damn rights to keep it for himself, call him sellfish or all sorts of names if you want, he doesn't owe you or the community anything, neither does Mafins.
> 
> So if you have issues with Menyoo, please find a new trainer that is up to date, because it's highly probable that Menyoo won't see any updates in a while.

EVERYTHING I SAID IN THAT MESSAGE IS RIGHT YOU RETARDED SHIT , SINCE WITH THAT MESSAGE I HAVE ALREADY EXPLAINED EVERYTHING THAT IS NEEDED TO BE DESCRIBED I WON'T WASTE MY TIME REPLYING ANYMORE TO YOU FUCKING SHIT , GO FUCKING KILL YOURSELF YOU EGOISTIC DICK

**Comment by frafflg** [comment]
> 
> 
> > > Are you a lawyer?
> > > You use it smoothly, why do those here complain?
> > 
> > 
> > don't mind that handicapped shit , he is only a motherfucking shit head with a face made of shit trowed here to bother us , your claims are completely legit , don't worry the trainer will be updated , it is a matter of common sense after all
> 
> **Thank you for your support.**

no problem my friend , i am definitively tired of all the luck of generosity and care that is affecting the world whatever we are talking about video games or whatever we are talking about something regarding life and death, as the two retarded assholes who dared to reply to us by saying all those bullshit demonstrated the problem is quite this cave man-kind of mentality, like:"oh yeah they\we aren't updating even if everyone need it since none can force us to do it and to give a damn, what are you gonna do\or what?", yup , this is the cave man-mentality that is the core of the problem , even the rocks know that none is forcing anyone to do something , but god fucking damn it , this behavior is what is ruining the world , this bullshit about not give a fuck about other and be egoistic and fucking lunatics , so yeah i also thanks you for accepting my support because the stupidity sometimes even affect those who have a problem you know , people that claim how unfair it is but at the same time not dare\approve when someone criticize the officialsource\person who is helping\supposed to help , so thanks again :D

**Comment by CptLeetsy** [comment]
Damn you're one salty boy.

Won't bother to argue further with you since you seems to be a 12yo used to have everything done when he wants, as he wants.
Sorry buddy but that's not how life work.

Good luck in your life with your behavior and anger issues, get that checked out btw.

Also Mafins, if you endup on that fustercluck of a thread, close it.

**Comment by ReadyP1ayerTwo** [comment]
> > > > > You persistently avoid updating.
> > > > 
> > > > 
> > > > What you fail to realize is mafins doesn't update it anymore himself. The community updates it. Infact, it got updated within a few recent months ago. Instead of people such as yourself complaining about bugs, You could make yourself useful and use your time more productively by fixing these bugs you people oh so endlessly complain about.
> > > > With that being said; i use menyoo all of the time with little to no issue.
> > > 
> > > 
> > > so there is another group or page on internet were the recent updates for this trainer are getting created , can you share the link here so we can access that page ?
> > 
> > 
> > Let me re-phrase; Recently enough considering the updates have been left to the community.
> > https://github.com/MAFINS/MenyooSP/releases
> 
> oh ok sorry sorry , i didn't understood that you are a motherfucking retarded shit , a miserable dick head that isn't of any help , this "do it yourself" is just fucking egoism , it is what assholes tell and do to others, if we could do that ourselves we would already did that , but since we aren't all hackers then we rightly ask to the person who created the trainer , you are the wrong one here , not the person who rightly demanded for this trainer to be updated , just shut the fuck up , oh right , i forgot that assholes like you think they are always right , so when someone insult them because they act egoistic and idiotic they get all surprised when they finally find someone who insult them , well here is the thing , your bullshit , your egoism is what caused you to get my insults , you are wrong and you keep insist despite this , mafins or manyoo created this trainer so he must update and keep developing it , your bullshit about the updates left to the community is just a less rude way to say "i don't give a fuck do it yourself" , is like you tried to make it sound like an encouragement , but you perfectly understand that none of us can update that trainer , so basically this is just not giving a damn about left some people with an unsolved problem , and to make thing even more insulting for us you even have the fucking nerve to complain , just shut the fuck up you useless dick head , fuck you , and keep in your fucked up mind that none is fully independent , not even you , if you need help for something that for some reasons can't achieve all by yourself then you need help , when that happen you will just deserve to get the same treatment that you inflicted on us here , dick , and i know that retarded assholes like you need things to be repeated many times otherwise you don't understand\pretend you didn't understood: I INSULTED YOU BECAUSE YOU INSULTED ME AND THE OTHER PERSON IN THIS DISCUSSION , IT WASN'T SOME RANDOM INSULTS , YOU FUCKING ASKED FOR THEM BECAUSE OF THE OFFENSIVE BULLSHIT YOU SAID , all clear ? it is clear enough for your handicapped little brain ? when you are rude to people , when you are a fucking pain in the ass expect at least this , got it you fuck ? this isn't about let anything for the community to do , this is just not giving a damn anymore about solve a problem , and yes in this case you can bet your ass that we want the problem to be solved and the update to come up in this page , this is what happen when the creator of an update care about it's creation so do not talk like all of a sudden it is something strange to expect , fucking asshole

tldr

**Comment by ReadyP1ayerTwo** [comment]
> > why do those here complain?
> 
> Probably because people like to complain and do nothing to solve the issue for themselves.
> People expect things to be spoon fed to them.

**Comment by frafflg** [comment]
> 
> 
> > > why do those here complain?
> > 
> > 
> > Probably because people like to complain and do nothing to solve the issue for themselves.
> > People expect things to be spoon fed to them.

yes , in this case we really expect this problem to be solved by itself , by the persons who created this mod to be precise , fucking retarded shit

**Comment by frafflg** [comment]
> 
> 
> > > > > > You persistently avoid updating.
> > > > > 
> > > > > 
> > > > > What you fail to realize is mafins doesn't update it anymore himself. The community updates it. Infact, it got updated within a few recent months ago. Instead of people such as yourself complaining about bugs, You could make yourself useful and use your time more productively by fixing these bugs you people oh so endlessly complain about.
> > > > > With that being said; i use menyoo all of the time with little to no issue.
> > > > 
> > > > 
> > > > so there is another group or page on internet were the recent updates for this trainer are getting created , can you share the link here so we can access that page ?
> > > 
> > > 
> > > Let me re-phrase; Recently enough considering the updates have been left to the community.
> > > https://github.com/MAFINS/MenyooSP/releases
> > 
> > 
> > oh ok sorry sorry , i didn't understood that you are a motherfucking retarded shit , a miserable dick head that isn't of any help , this "do it yourself" is just fucking egoism , it is what assholes tell and do to others, if we could do that ourselves we would already did that , but since we aren't all hackers then we rightly ask to the person who created the trainer , you are the wrong one here , not the person who rightly demanded for this trainer to be updated , just shut the fuck up , oh right , i forgot that assholes like you think they are always right , so when someone insult them because they act egoistic and idiotic they get all surprised when they finally find someone who insult them , well here is the thing , your bullshit , your egoism is what caused you to get my insults , you are wrong and you keep insist despite this , mafins or manyoo created this trainer so he must update and keep developing it , your bullshit about the updates left to the community is just a less rude way to say "i don't give a fuck do it yourself" , is like you tried to make it sound like an encouragement , but you perfectly understand that none of us can update that trainer , so basically this is just not giving a damn about left some people with an unsolved problem , and to make thing even more insulting for us you even have the fucking nerve to complain , just shut the fuck up you useless dick head , fuck you , and keep in your fucked up mind that none is fully independent , not even you , if you need help for something that for some reasons can't achieve all by yourself then you need help , when that happen you will just deserve to get the same treatment that you inflicted on us here , dick , and i know that retarded assholes like you need things to be repeated many times otherwise you don't understand\pretend you didn't understood: I INSULTED YOU BECAUSE YOU INSULTED ME AND THE OTHER PERSON IN THIS DISCUSSION , IT WASN'T SOME RANDOM INSULTS , YOU FUCKING ASKED FOR THEM BECAUSE OF THE OFFENSIVE BULLSHIT YOU SAID , all clear ? it is clear enough for your handicapped little brain ? when you are rude to people , when you are a fucking pain in the ass expect at least this , got it you fuck ? this isn't about let anything for the community to do , this is just not giving a damn anymore about solve a problem , and yes in this case you can bet your ass that we want the problem to be solved and the update to come up in this page , this is what happen when the creator of an update care about it's creation so do not talk like all of a sudden it is something strange to expect , fucking asshole
> 
> tldr

shut up you retarded shit

**Comment by ReadyP1ayerTwo** [comment]
> shut up you retarded shit, you have read it , it tell how you are fucking wrong and how that damn trainer must be fucking updated from the person who created it , dick , handicapped dick

If it's that big of an issue for you to go getting bent all out of shape, you're more than welcome to try fixing it yourself! Have a great day!

**Comment by frafflg** [comment]
> 
> 
> > shut up you retarded shit, you have read it , it tell how you are fucking wrong and how that damn trainer must be fucking updated from the person who created it , dick , handicapped dick
> 
> If it's that big of an issue for you to go getting bent all out of shape, you're more than welcome to try fixing it yourself! Have a great day!

no you retarded shit , the people who made that trainer must be the ones that fix it , keep that in mind you dick , go fucking kill yourself , or at least cover that damn photo of you , it sucks , go fucking kill yourself along with the assholes who appreciated that comment of yours

**Comment by frafflg** [comment]
> 
> 
> Damn you're one salty boy.
> 
> Won't bother to argue further with you since you seems to be a 12yo used to have everything done when he wants, as he wants.
> Sorry buddy but that's not how life work.
> 
> Good luck in your life with your behavior and anger issues, get that checked out btw.
> 
> Also Mafins, if you endup on that fustercluck of a thread, close it.

you are an handicapped shit , whoever make that trainer must fucking update it  , not acting like it never did anything and not give a damn about , but hey , you and the other shit are acting like it is ok for you to be egoistic and not give a damn , you will pay for this , no doubt about it

**Comment by frafflg** [comment]
> 
> 
> Then the Menyoo Programmer has to announce that it's over.
> People here should not be in expectation.
> Updates by the community are not enough.
> It contains many errors.
> Therefore, shouldn't the programmer say that I will no longer support it?

don't worry , that bullshit about "do it yourself" is because they think they are so clever , to look like they aren't actually just letting us alone with this problem , claim that we should do something on our own , yeah right , like we never wanted to actually do something to fucking solve the problem , that bullshit only have to be ignored , also with your post and your comments demonstrated something very very important: those handicapped shits are like machines alimented with insults, they don't even care to reply unless they are insulted and yet again they had the nerve to complain when people insult them , so those dicks just asked for be insulted , infact i am sure that they have read the previous requests for the updates , but , there were no insults , no provocations , so they didn't care about reply

**Comment by ZatarraTR** [comment]
> > Then the Menyoo Programmer has to announce that it's over.
> > People here should not be in expectation.
> > Updates by the community are not enough.
> > It contains many errors.
> > Therefore, shouldn't the programmer say that I will no longer support it?
> 
> don't worry , that bullshit about "do it yourself" is because they think they are so clever , to look like they aren't actually just letting us alone with this problem , claim that we should do something on our own , yeah right , like we never wanted to actually do something to fucking solve the problem , that bullshit only have to be ignored , also with your post and your comments demonstrated something very very important: those handicapped shits are like machines alimented with insults, they don't even care to reply unless they are insulted and yet again they had the nerve to complain when people insult them , so those dicks just asked for be insulted , infact i am sure that they have read the previous requests for the updates , but , there were no insults , no provocations , so they didn't care about reply

**We are right. We have been waiting for months. I am with you until the end.**

**Comment by frafflg** [comment]
> 
> 
> > > Then the Menyoo Programmer has to announce that it's over.
> > > People here should not be in expectation.
> > > Updates by the community are not enough.
> > > It contains many errors.
> > > Therefore, shouldn't the programmer say that I will no longer support it?
> > 
> > 
> > don't worry , that bullshit about "do it yourself" is because they think they are so clever , to look like they aren't actually just letting us alone with this problem , claim that we should do something on our own , yeah right , like we never wanted to actually do something to fucking solve the problem , that bullshit only have to be ignored , also with your post and your comments demonstrated something very very important: those handicapped shits are like machines alimented with insults, they don't even care to reply unless they are insulted and yet again they had the nerve to complain when people insult them , so those dicks just asked for be insulted , infact i am sure that they have read the previous requests for the updates , but , there were no insults , no provocations , so they didn't care about reply
> 
> **We are right. We have been waiting for months. I am with you until the end.**

many thanks :D

**Comment by frafflg** [comment]
@CptLeetsy go fucking kill yourself along with that emoticon under my comment you fucking shit

**Comment by porcodio1234567** [comment]
> > > > > > You persistently avoid updating.
> > > > > 
> > > > > 
> > > > > What you fail to realize is mafins doesn't update it anymore himself. The community updates it. Infact, it got updated within a few recent months ago. Instead of people such as yourself complaining about bugs, You could make yourself useful and use your time more productively by fixing these bugs you people oh so endlessly complain about.
> > > > > With that being said; i use menyoo all of the time with little to no issue.
> > > > 
> > > > 
> > > > so there is another group or page on internet were the recent updates for this trainer are getting created , can you share the link here so we can access that page ?
> > > 
> > > 
> > > Let me re-phrase; Recently enough considering the updates have been left to the community.
> > > https://github.com/MAFINS/MenyooSP/releases
> > 
> > 
> > oh ok sorry sorry , i didn't understood that you are a motherfucking retarded shit , a miserable dick head that isn't of any help , this "do it yourself" is just fucking egoism , it is what assholes tell and do to others, if we could do that ourselves we would already did that , but since we aren't all hackers then we rightly ask to the person who created the trainer , you are the wrong one here , not the person who rightly demanded for this trainer to be updated , just shut the fuck up , oh right , i forgot that assholes like you think they are always right , so when someone insult them because they act egoistic and idiotic they get all surprised when they finally find someone who insult them , well here is the thing , your bullshit , your egoism is what caused you to get my insults , you are wrong and you keep insist despite this , mafins or manyoo created this trainer so he must update and keep developing it , your bullshit about the updates left to the community is just a less rude way to say "i don't give a fuck do it yourself" , is like you tried to make it sound like an encouragement , but you perfectly understand that none of us can update that trainer , so basically this is just not giving a damn about left some people with an unsolved problem , and to make thing even more insulting for us you even have the fucking nerve to complain , just shut the fuck up you useless dick head , fuck you , and keep in your fucked up mind that none is fully independent , not even you , if you need help for something that for some reasons can't achieve all by yourself then you need help , when that happen you will just deserve to get the same treatment that you inflicted on us here , dick , and i know that retarded assholes like you need things to be repeated many times otherwise you don't understand\pretend you didn't understood: I INSULTED YOU BECAUSE YOU INSULTED ME AND THE OTHER PERSON IN THIS DISCUSSION , IT WASN'T SOME RANDOM INSULTS , YOU FUCKING ASKED FOR THEM BECAUSE OF THE OFFENSIVE BULLSHIT YOU SAID , all clear ? it is clear enough for your handicapped little brain ? when you are rude to people , when you are a fucking pain in the ass expect at least this , got it you fuck ? this isn't about let anything for the community to do , this is just not giving a damn anymore about solve a problem , and yes in this case you can bet your ass that we want the problem to be solved and the update to come up in this page , this is what happen when the creator of an update care about it's creation so do not talk like all of a sudden it is something strange to expect , fucking asshole
> 
> tldr

go fucking kill yourself , you retarded shit
